runtime/queries/gosum/highlights.scm

```diff
@@ -8,11 +8,11 @@
 ] @keyword
 
 
-(module_path) @string @text.uri
+(module_path) @string.special.path
```

Comment:
Is this not a URL?

Reply:
I don't know; from what I could tell it's a local path. But happy to change if you know better!

Reply:
Seems to be a URL.